server/statistics/util.go

```diff
@@ -27,3 +30,13 @@ const (
 func storeTag(id uint64) string {
 	return fmt.Sprintf("store-%d", id)
 }
+
+func GetLoads(r *core.RegionInfo) []float64 {
```

Comment:
Is this function belong to region?

Reply:
yes, It is placed here to avoid circular references between core and statistic

Reply:
I think it can be removed now?

Reply:
done